src/main/java/net/pms/network/mediaserver/handlers/ApiHandler.java

```diff
@@ -32,46 +33,47 @@ public ApiHandler() {
 	 * @param output Response to request
 	 * @param event The request
 	 * @param uri
+	 *
+	 * @return response String
 	 */
-	public void handleApiRequest(HttpMethod method, String content, HttpResponse output, String uri, MessageEvent event) {
-		output.headers().set(HttpHeaders.Names.CONTENT_LENGTH, "0");
-		output.setStatus(HttpResponseStatus.NO_CONTENT);
-
+	public String handleApiRequest(HttpMethod method, String content, HttpResponse output, String uri, MessageEvent event) {
 		String serverApiKey = PMS.getConfiguration().getApiKey();
 		if (serverApiKey.length() < 12) {
-			LOGGER.warn("Weak server API key configured. UMS.conf api_key should have at least 12 digests.");
+			LOGGER.warn("Weak server API key configured. UMS.conf api_key should have at least 12 characters.");
 			output.setStatus(HttpResponseStatus.SERVICE_UNAVAILABLE);
-			return;
+			return "Weak or no server API key configured. UMS.conf api_key should have at least 12 characters.";
```

Comment:
Now that the return string is being returned with the response, we should make it translatable